include/pybind11/stl_binders.h

```diff
@@ -0,0 +1,184 @@
+/*
+    pybind11/std_binders.h: Convenience wrapper functions for STL containers with C++ like interface
+
+    Copyright (c) 2016 Sergey Lyskov
+
+    All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file.
+*/
+
+#ifndef _INCLUDED_std_binders_h_
+#define _INCLUDED_std_binders_h_
+
+#include "common.h"
+#include "operators.h"
+
+#include <type_traits>
+#include <utility>
+#include <algorithm>
+#include <sstream>
+
+
+NAMESPACE_BEGIN(pybind11)
+
+template<typename T>
```

Comment:
I think that would be a pretty broken implementation of the underlying type, not something that we need to worry about in pybind11.